config/config.toml

```diff
@@ -79,9 +79,18 @@ backupCheckSleepDurationIns = 5
 backupRowsBatchSize = 10000
 
```

Comment:
Add a comment mentioning ->
By default, only gateway tables are backed up. If you want to back up only failed events instead of backing up the complete table, set abortedOnly to true.

Reply:
added the comments as example

Reply:
In the config.toml, lets make the following blocks complete? I mean "failedOnly" is missing in "JobsDB.backup.gw" and "pathPrefix" is missing in the other two blocks.

[JobsDB.backup.gw]
enabled=true
pathPrefix=""

[JobsDB.backup.rt]
enabled=false
failedOnly=false

[JobsDB.backup.batch_rt]
enabled=false
failedOnly=false